src/draw/sdl/lv_draw_sdl.h

```diff
@@ -44,6 +44,7 @@ typedef struct {
      * Render for display driver
      */
     SDL_Renderer * renderer;
+    void * userdata;
```

Comment:
For consistency wit other parts of LVGL:

```suggestion
    void * user_data;
```